source/compatibility.tex

```diff
@@ -1851,7 +1852,7 @@
 to ``array of \tcode{const wchar_t}''.
 \rationale
 This avoids calling an inappropriate overloaded function,
-which might expect to be able to modify its argument.
+which would possibly attempt to modify its argument.
```

Comment:
This sounds like any such inappropriate function has a chance of making such a modification.  Maybe say "can be expected"?  It's a bit of a cheat, since we're not really saying that it's _possible_ for a person to expect the function to do something, but rather that it's a _reasonable_ expectation, but it's an allowed verb.

Reply:
I'll leave this for the future.